packages/sit-onyx/src/components/OnyxNavItem/future/OnyxNavItem.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+const props = defineProps<{ active?: boolean }>();
+</script>
+<template>
+  <li
```

Comment:
should it really be a li?
in the DOM, we now have li inside li
![image](https://github.com/SchwarzIT/onyx/assets/151019977/92c23478-7f35-4d33-b4a6-3f7d33eb572c)

Reply:
I refactored the nav item and we are using the list item internally which means this is not the structure any more :)